docs/source/INSTRUCTORS.md

```diff
@@ -0,0 +1,91 @@
+
+For the project improvements and assignments, meeting with the students 2 times a day will give you likely observations regarding how a student is progressing with their needs and executions.
+
+### What am I expected to do during a clinic/Q&A call?
+
+During the Q&A hours in weeks 1 and 2, you are expected to catch up with the students regarding their module assignments and queries. 
+
+
+During the clinic hours in weeks 3 and 4, you will track the student's progress with their project execution and share your feedback if they have any specific queries.
+
+Please do not forget you are not expected to provide definite help or the correct answer. You are there to give feedback and point to the possible resources that might help them. 
+
+
+### What should I do if a student is not progressing with their project?
+
+Sometimes some students are stuck with an idea, choosing the right tool, or finding the data for their project. In some cases, they might also be hesitant to raise their questions. 
+
+The best way to help those students would be to ask what aspect of their project is preventing them from progressing and give feedback to them to narrow down their project scope to facilitate execution.
+
+Please remember you are there to provide means for students to progress rather than tackling the question together. You are not expected to boggle down the code or execute a task for them, or not to micro-manage their progress but to give them insight and guidance regarding how they can move forward in a possible way. 
+
+
+
+### What should I do if a student needs to complete module assignments?
+
+Similar to project executions, sometimes students might struggle with finishing the modules and the assignments due to technical struggles or time concerns because of other commitments they have. 
+
+In such cases, reach students to hear the reason that lags them behind. If it is a technical problem request them to raise their problem in the Discord channels to get more help. Please contact the local hub organizers for their advice if it is due to the need for more time. 
+
+Please remind yourself that strict requirements are especially expected for the students who take the course as a credited course. 
+
+### What should I do if a student claims they are already proficient in some modules and would like to be excluded from finishing the module issued as those modules are completed?
+
+Ask them for previous work that might show their proficiency in the modules. For eg., some students might be competent in using git/GitHub already, and they might have a repository and share the link with you. 
+
+However, please remember to keep the engagement and requests on kind terms rather than an obligatory ones.
```

Comment:
I don't know if it makes sense to have this here, we could just have a general line about communication with students? Also 'obligatory terms' is not super clear